### ISSUE TITLE
Ability to see factionmates' health

### ISSUE THREAD

**Issue by Wingysam** [GalaxyRoblox/Issues-Public#29]
You should maybe be able to see the current health stats of people in the same faction as you. Could also make it for your team but that has crossteam-benefitting drawbacks.

Blocked by #5

**Comment by PataNautic** [comment]
Probably not a good idea until we have a better grasp on how to tackle crossteaming; I see this being easily abused to pick out kills. Most team-based mechanics should be out of the question until we are able to deal with crossteaming in its current form.

**Comment by cswimr** [comment]
Think there's a good distinction to be made here, yname stated "in the same faction" and later stated "for your team", implying that the initial change would only show health for factions and not player teams. I.E. If you're in the LS faction, but there are other LS members in other teams in the server, you could see health of any people in the LS faction, regardless of their player team. I don't think this is a terrible idea, but the "for your team" part would definitely have to come after crossteaming is addressed.

**Comment by Wingysam** [comment]
Yeah @PataNautic I agree, we can't just ship this for seeing the hp of randoms in your team yet. Do you see any problems with [player factions](https://github.com/GalaxyRoblox/Issues-Public/issues/5) allowing you to see your friends' hp, even if they're in other teams?

**Comment by PataNautic** [comment]
I think the information should be obfuscated unless they are both on the same team and in the same faction. I'm iffy about that information being available between teams, but if they're on the same team, it should be OK

**Comment by Wingysam** [comment]
Yeah that's fair enough, cross-team hp viewing probably encourages cross-teaming.

**Comment by PataNautic** [comment]
Blocked by #5 (for reference)
